test/bin/scenario.sh

```diff
@@ -1096,6 +1063,74 @@ check_dependencies() {
     fi
 }
 
+create_input_variables_file() {
+    local -r vmname="$1"
+    local variable_file="${SCENARIO_INFO_DIR}/${SCENARIO}/variables.yaml"
+    echo "[INFO] Writing variables to ${variable_file}" >&2
+    mkdir -p "$(dirname "${variable_file}")"
+
+    cat - <<EOF > "${variable_file}"
+VM_IP: ${vm_ip}
+API_PORT: ${api_port}
+LB_PORT: ${lb_port}
+USHIFT_HOST: ${vm_ip}
+USHIFT_USER: redhat
+SSH_PRIV_KEY: "${SSH_PRIVATE_KEY:-}"
+SSH_PORT: ${ssh_port}
+EOF
+    echo "${variable_file}"  # return path
```

Comment:
This would produce an extra message in the regular flow. 
Should we move the variable to global scope so that it can be accessed outside of the function?